source/_integrations/starlink.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: Starlink
+description: Monitor Starlink satellite status from home Assistant
+  - Binary Sensor
+  - Sensor
+ha_release: 
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@pacorain'
+ha_domain: starlink
+ha_platforms:
+  - binary_sensor
+  - sensor
+---
+The `starlink` platform consumes data from your [Starlink sattelite](https://www.starlink.com/) into Home Assistant sensors.
+
+{% include integrations/config_flow.md %}
+
+{% configuration %}
```

Comment:
Use the basic configuration instead, `configuration` is meant for YAML-based integrations.